client/question/services/ConversationManagerServiceTestUtils.js

```diff
@@ -129,11 +112,8 @@ tie.factory('ExpectedFeedbackObjectFactory', [function() {
   /**
    * Creates and returns an ExpectedFeedback object.
    *
-   * @param {string|null} firstFeedbackParagraphContent The expected content of
-   *   the first feedback paragraph, or null if we do not need to check this.
-   * @param {string|null} secondFeedbackParagraphContent The expected content
-   *   of the second feedback paragraph, or null if we do not need to check
-   *   this.
+   * @param {Array<string>|null} expectedFeedbackParagraphs The content of
+   *   expected feedback paragraphs.
```

Comment:
Ditto re explaining the significance of null.

Reply:
Done in next commit.